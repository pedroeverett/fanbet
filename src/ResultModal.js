import React from 'react'
import { Button, Header, Modal } from 'semantic-ui-react'

const ResultModal = ({onClick, open, fppg, message, buttonLabel}) => (
  <Modal open={open} centered={false}>
    <Header icon='clipboard' content='Your result this time was...' />
    <Modal.Content>
      <Modal.Description>
        <Header>{message}</Header>
        <h4>
          Your player FPPG score is: {fppg}
        </h4>
      </Modal.Description>
    </Modal.Content>
    <Modal.Actions>
      <Button color='green' inverted onClick={onClick}>
        {buttonLabel}
      </Button>
    </Modal.Actions>
  </Modal>
)

export default ResultModal;