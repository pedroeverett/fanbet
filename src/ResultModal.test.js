import React from 'react';
import { shallow } from 'enzyme';
import ResultModal from './ResultModal';

describe('ResultModal', () => {
  const component = shallow(<ResultModal />);
//   const { player } = component.props
  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });

//   it('should get props correctly', () => {
//     expect(player.first_name).toBe('pedro');
//   });

//   it('should contain 1 div element', () => {
//     expect(component.find('div').length).toBe(1);
//   })

});