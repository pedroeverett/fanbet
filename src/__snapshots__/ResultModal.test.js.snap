// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultModal should render correctly 1`] = `
<Modal
  centered={false}
  closeOnDimmerClick={true}
  closeOnDocumentClick={false}
  dimmer={true}
  eventPool="Modal"
>
  <Header
    content="Your result this time was..."
    icon="clipboard"
  />
  <ModalContent>
    <ModalDescription>
      <Header />
      <h4>
        Your player FPPG score is: 
      </h4>
    </ModalDescription>
  </ModalContent>
  <ModalActions>
    <Button
      as="button"
      color="green"
      inverted={true}
    />
  </ModalActions>
</Modal>
`;
